cognite/client/_api/extractionpipelines.py

```diff
@@ -18,6 +20,11 @@
 class ExtractionPipelinesAPI(APIClient):
     _RESOURCE_PATH = "/extpipes"
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        self.runs = ExtractionPipelineRunsAPI(*args, **kwargs)
```

Comment:
Ah, you already had one of these. Then I would suggest making

`CogniteClient::extraction_pipeline_runs` raise a DeprecationWarning stating that it will be removed in the next major version, and that the user should use `CogniteClient::extraction_pipeline.runs` instead.

Reply:
Did the same is in the PR you linked, but deprecated in v6 like that one, since it looks like v5 is coming quite soon, not much of a warning for users if we really remove it then.